server/database/fixtures/UserSeeder.js

```diff
@@ -11,31 +11,31 @@ class UserSeeder extends AbstractSeeder {
         firstname: "curtis",
         lastname: "dakouri",
         mail: "curtis.dakouri@gmail.com",
-        hashed_password: "kirikou.24@",
+        password: "kirikou.24@",
```

Comment:
password in your seeder must be hashed otherwise it will be impossible to connect with this users
import argon2 and hash password here